toolchain/sem_ir/formatter.cpp

```diff
@@ -829,6 +837,14 @@ class FormatterImpl {
     }
   }
 
+  auto FormatInstRHS(GenericInterfaceType inst) -> void {
+    if (inst.enclosing_specific_id.is_valid()) {
```

Comment:
Minor suggestion, comparing with FunctionType, ClassType, and InterfaceType, maybe add a helper:

```
template<typename EntityId>
auto FormatInstRHSForEntity(EntityId entity_id, SpecificId specific_id) -> void {
  if (specific_id.is_valid()) {
    FormatArgs(entity_id, specific_id);
  } else {
    FormatArgs(entity_id);
  }
}
```

Reply:
Yeah, we have a bunch of instructions like this. I'll send a follow-up PR for this.